src/subscription_manager/repofile.py

```diff
@@ -583,6 +583,8 @@ def read_zypp_conf(self):
         zypp_cfg.read(self.ZYPP_RHSM_PLUGIN_CONFIG_FILE)
         if zypp_cfg.has_option('rhsm-plugin', 'gpgcheck'):
             self.gpgcheck = zypp_cfg.getboolean('rhsm-plugin', 'gpgcheck')
+        if zypp_cfg.has_option('rhsm-plugin', 'repo_gpgcheck'):
+            self.repo_gpgcheck = zypp_cfg.getboolean('rhsm-plugin', 'repo_gpgcheck')
```

Comment:
Other configuration options are initialized in `__init__()` method. Please do it for new attribute `self.repo_gpgcheck` too and initialize it to default value (`False` in this case)

Reply:
done